src/index.js

```diff
@@ -145,4 +145,13 @@ export default class BreakLine {
       title: 'Break Line',
     };
   }
+
+  /**
+   * Returns true to notify the core that read-only mode is supported
+   *
+   * @return {boolean}
+   */
+  static get isReadOnlySupported() {
```

Comment:
@MrJuanGaviriaK this is ok 👌   the tool will notify the `editor.js` **core** that the `readOnly` mode is supported. 

I think there is a missing piece, the `render()` method should not perform an action when this mode is active. 

What do you think?